appium/tests/specs/composeEmail/SelectRecipientByName.spec.ts

```diff
@@ -15,12 +15,18 @@ describe('COMPOSE EMAIL: ', () => {
 
     it('user is able to select recipient from contact list using contact name', () => {
 
-        const contactEmail = CommonData.contact.email;
-        const contactName = CommonData.contact.name;
+        const firstContactEmail = CommonData.contact.email;
+        const firstContactName = CommonData.contact.name;
+        const firstContactItemName = 'Dmitry at FlowCrypt';
```

Comment:
For some reason contact name and Contacts screen item name are different values

Reply:
That may be possible. Eg, maybe the contact search name comes from Google Contacts but this name comes from PGP key user id.